### PR TITLE
[release-v1.15] Run `make generate-release`

### PR DESCRIPTION
**Pull request by serverless-qe**
Run `make generate-release` using openshift-knative/hack.

### CODE REVIEW

File: openshift/ci-operator/knative-images/kn/Dockerfile
Comment (creydr) [inline openshift/ci-operator/knative-images/kn/Dockerfile:1]: @dsimansk this was the result of `make generate-release` is this change intended?

Comment (dsimansk) [reply openshift/ci-operator/knative-images/kn/Dockerfile:1]: Thanks, added that to a patch. However, I'll think about more maintainable approach.

**Comment by dsimansk** [comment]
/override "Red Hat Konflux / kn-client-cli-artifacts-115-on-pull-request"
/override "Red Hat Konflux / kn-client-kn-115-on-pull-request"
/override "Red Hat Konflux / kn-client-test-grpc-ping-115-on-pull-request"
/override "Red Hat Konflux / kn-client-test-helloworld-115-on-pull-request"
/override "Red Hat Konflux / kn-client-test-servingcontainer-115-on-pull-request"
/override "Red Hat Konflux / kn-client-test-sidecarcontainer-115-on-pull-request"
